frontend/src/pages/ModelTester/Dashboard2/Dashboard2.jsx

```diff
@@ -188,7 +186,7 @@ const Dashboard2 = () => {
 
       <div className="chart-section">
         {Object.keys(graphData).length > 0 && (
-          <ChartComponent2 chartData={graphData} />
+          <ChartComponent2 chartData={graphData} generationalResults={generationResults} />
```

Comment:
can you explain what generationalResults const do?

Reply:
Its a variable for storing the tooltip hover data for each bar of the graph